src/app/parcels/ParcelsPage.tsx

```diff
@@ -709,37 +732,54 @@ const ParcelsPage: React.FC<{}> = () => {
                         pointerOnHover={true}
                     />
                 </TableSurface>
+                    <Modal
```

Comment:
i don't think the indentation is right - try linting

Reply:
I've rerun lint_fix which seems to have updated this correctly now